### PR TITLE
⭐️ Support AWS CloudShell install in install.sh

### PR DESCRIPTION
**Pull request by jaym**
Detects if script is being run on AWS CloudShell instance. When this is detected, the installer installs Mondoo to `$HOME/.local/bin`

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -97,6 +97,8 @@ if [ "$DISTRIBUTION" = "Darwin" ]; then
   OS="macOS"
 elif [ -f /etc/debian_version ] || [ "$DISTRIBUTION" == "Debian" ] || [ "$DISTRIBUTION" == "Ubuntu" ]; then
   OS="Debian"
+elif [ "$AWS_EXECUTION_ENV" == "CloudShell" ]; then
+  OS="CloudShell"
```
Comment (chris-rock) [inline install.sh:101]: should we name this "AWSCloudShell" since we probably will add support for GCP and Azure going forward

File: install.sh [hunk 2]
```diff
@@ -201,18 +203,19 @@ install_portable() {
   URL="https://releases.mondoo.io/mondoo/${MONDOO_LATEST_VERSION}/${FILE}"
 
   echo "Downloading the latest version of Mondoo from: $URL"
-  curl -O "${URL}"
-
-  tar xf "${FILE}"
+  curl "${URL}" | tar xz
```
Comment (chris-rock) [inline install.sh:206]: +1

File: install.sh [hunk 3]
```diff
@@ -201,18 +203,19 @@ install_portable() {
...
   detect_portable
   if [ -z "$MONDOO_EXECUTABLE" ]; then
     red "We could not find the 'mondoo' executable in the present working directory."
     exit 1
   fi
 
-  purple_bold "We installed a portable version of mondoo to the present working directory."
-  echo "You can run Mondoo via: ${MONDOO_CMD}"
+  purple_bold "We installed a portable version of mondoo to $PWD"
+  if [[ ":$PATH:" == ":$PWD:" ]]; then
```
Comment (chris-rock) [inline install.sh:215]: This has not shown the text to me:

```
* Installing Mondoo via tar

* Installing Mondoo to /home/cloudshell-user/.local/bin
Downloading the latest version of Mondoo from: https://releases.mondoo.io/mondoo/5.28.0/mondoo_5.28.0_linux_amd64.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 38.6M  100 38.6M    0     0  57.1M      0 --:--:-- --:--:-- --:--:-- 57.1M
We installed a portable version of mondoo to /home/cloudshell-user/.local/bin
Mondoo installation completed.
```

**Review by chris-rock — COMMENTED** [review]
Super great @jaym I tested it quickly and it just works, also update works too. So cool. I found an issue where the export line was not displayed. I am not sure if this is by intention or not.

File: install.sh (hunk 3 above)
Comment (jaym) [reply install.sh:215]: A new shell has `$HOME/.local/bin` so the user has to do nothing extra
```
[cloudshell-user@ip-10-0-81-168 ~]$ echo $PATH
/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/home/cloudshell-user/.local/bin:/home/cloudshell-user/bin
[cloudshell-user@ip-10-0-81-168 ~]$
```

**Review by chris-rock — APPROVED** [review]
Great improvement!

**Pull request merged**